site/soc/software/charts/osh/openstack-keystone/keystone.yaml

```diff
@@ -30,4 +30,57 @@ data:
     pod:
       replicas:
         api: 1
+      security_context:
+        keystone:
+          pod:
+            runAsUser: 0
+          container:
+            keystone_api:
+              readOnlyRootFilesystem: false
+    conf:
```

Comment:
this is still kind of terrible. Why not take advantage of the ansible templating capabilities and use this file as a template and *include* the other file with the overrides? 

currently if we change something on our templates and you are not aware, airship deployment will fail. If you link both of this files by including the osh overrides in here, then if we change something it will be fixed automatically, making it easier to maintain, cleaner and much clear where this overrides are coming from.

I just dont understand why are we doing manual work on each template when it should be left for the machines, which are really good at menial tasks :)

Reply:
You know that we want to push such change to upstream and then we would not need to carry it with us...

Reply:
that fix would still work once we remove it from osh as airship would not need it anymore thus it would not be automatically included here. Its about making it easier to maintain...

Reply:
@jsuchome do you have the link to the upstream change to include these? yes, definitely this should go upstream.

Reply:
I looked at a couple of patches using the value_overrides folder. Good idea, I am curious if the upstream osh team has talked to upstream airship team how the new pattern will be consumed into Airship?

Reply:
we likely need to separate the values_overrides into a different commit pending how the upstream airship plans to consume the new pattern.

Reply:
Good point, but I do not have an idea how Airship is deploying helm. What do you think, is it possible to use the arguments pointing to values_overrides directories?

Reply:
That needs an upstream discussion the best approach to handle release/distro flavor overrides in Airship w/o having to provide full distro overrides like this PR. I have some ideas but want to talk to the att engineers at ptg next week. For tech preview, I think we should merge this PR for now.

Reply:
:+1: